### PR TITLE
app-laptop/i8kutils: Upgrade to v1.42

### PR DESCRIPTION
**Pull request by 10ne1**
Updated request based on feedback from @jlec

### CODE REVIEW

File: app-laptop/i8kutils/i8kutils-1.42.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+SRC_URI="https://launchpad.net/i8kutils/trunk/${PV}/+download/${P/-/_}.tar.xz"
+
+S="${WORKDIR}/${PN}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="-* amd64 x86"
+IUSE="examples tk"
+
+DEPEND="tk? ( dev-lang/tk )"
```
Comment (jlec) [inline app-laptop/i8kutils/i8kutils-1.42.ebuild]: Missing SLOT here.

File: app-laptop/i8kutils/i8kutils-1.42.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/app-laptop/i8kutils/i8kutils-1.42.ebuild,v 1.3 2015/05/17 15:03:53 Ionel A $
```
Comment (jlec) [inline app-laptop/i8kutils/i8kutils-1.42.ebuild]: convert this to ```# $Id$```

**Comment by jlec** [comment]
@idella Could you please take a look into this?

**Comment by idella** [comment]
certainly, on it

**Comment by idella** [comment]
Not sure where to go with this. The patch provided does not actually have the proxy-maintainers herd and is absent of an email address. A stray pull request stands as a leftover obstacle to prevent us from retiring this gentoo overlay otherwise no longer needed.

**Comment by 10ne1** [comment]
I've updated the metadata and submitted the pull request to gentoo/gentoo. I'll close this pull request to not impede the retirement of this repo.

**Comment by mrueg** [comment]
Thanks! :+1:

**Pull request closed without merging**